### DISCUSSION TITLE
Bait List

### DISCUSSION THREAD

**Discussion by markjsosnowski** [markjsosnowski/OceanFishin#4] [General]
The bait list that Ocean Fishin' uses to recommend bait is found at https://markjsosnowski.github.io/FFXIV/bait.json

These recommendations are based on

-  [Zeke's Ocean Fishing Guidebook](https://docs.google.com/spreadsheets/d/17A_IIlSO0wWmn8I3-mrH6JRok0ZIxiNFaDH2MhN63cI/ "Google Sheets")
-  [S’yahn Tia's Ocean Fishing Spreadsheet](https://docs.google.com/spreadsheets/d/1brCfvmSdYl7RcY9lkgm_ds8uaFqq7qaxOOz-5BfHuuk/ "Google Sheets")
- Advice from the Fisherman’s Horizon Discord server.

Some more popular sources are considered slightly outdated by regular ocean fishers. Some of the differences that Ocean Fishin' uses that may conflict with popular sources include:

- Galadion Bay: Krill usually is a better starting choice than Plump Worm in most weathers.
- The Northern Straight at Sunset and Night: Krill is a safer choice over Ragworm during the spectral current.

**Please put all suggestions and questions regarding the bait list in this thread.**

**Comment by genoism** [comment]
Any chance you could please update this for 6.1? Many thanks!

**Comment by SomeCookie07** [comment]
Please an update ♥

**Comment by feralproto** [comment]
An update would be fantastic <3 Unable to find this on Dalamud at all. Only conclusion I can come to is it is outdated.
